py/desispec/workflow/desi_proc_funcs.py

```diff
@@ -73,6 +73,9 @@ def get_shared_desi_proc_parser():
     parser.add_argument("--timingfile", type=str, help='save runtime info to this json file; augment if pre-existing')
     parser.add_argument("--no-xtalk", action="store_true", help='diable fiber crosstalk correction')
     parser.add_argument("--system-name", type=str, help='Batch system name (cori-haswell, perlmutter-gpu, ...)')
+    parser.add_argument("--extract-size", type=int, default=22, help="Size to use for GPU extract subcomm")
+    parser.add_argument("--gpuspecter", action="store_true", help="Use GPU specter")
+    parser.add_argument("--gpuextract", action="store_true", help="Use GPU extraction")
```

Comment:
For the record (no action needed): gpu_specter includes a CPU-based code path which is faster than the original specter, even without using GPUs.  i.e. it could make sense to use `--gpuspecter` without `--gpuextract` on a CPU-only node.

Reply:
Agreed. Will be interesting to try this on Cori, for example.